proto/cusf/mainchain/v1/wallet.proto

```diff
@@ -16,6 +16,8 @@ service WalletService {
     returns (CreateDepositTransactionResponse);
   rpc CreateNewAddress(CreateNewAddressRequest)
     returns (CreateNewAddressResponse);
+  rpc CreateSidechainProposal(CreateSidechainProposalRequest)
+    returns (CreateSidechainProposalResponse);
```

Comment:
Would be nice with a comment here (and the other RPCs, tbh) that further explain what's going on. E.g. something like `Creates a new sidechain proposal (M1 in BIP300) and persist to the local database for further processing. Sidechain proposals must be included in the coinbase transaction of a newly mined block, so this proposal is not active until the wallet has been able to generate a new block`